tests/flow/async_iteration/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -134,9 +134,11 @@ declare var gen: AsyncGenerator<void, string, void>;
 
 // You can pass whatever you like to return, it doesn\'t need to be related to
 // the AsyncGenerator\'s return type
-gen.return(0).then(result => {
-  (result.value: void); // error: string | number ~> void
-});
+gen
+  .return(0)
+  .then(result => {
+    (result.value: void); // error: string | number ~> void
+  });
```

Comment:
Yeah, this is the kind of breaking above that I intentionally tried to avoid. I think it looks better if we allow the last function to wrap around instead.

Personally I was fine with the old behavior, and I hate the testing frameworks that do `.a.lookup.for.every.single.word`. But that doesn't mean that pattern doesn't exist. I haven't studied your code enough to suggest any tweaks but I'll take a look.